go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containers/conmon
 
-go 1.18
+go 1.19
```

Comment:
I think switching to go 1.22 is reasonable, right?

Reply:
agreed!

Reply:
While at it, I think we should get off from the github.com/pkg/errors package. Some of the @saschagrunert lint magic could help here a lot to identify old and obsolete things. :smile: